tools/linter_lib/custom_check.py

```diff
@@ -489,16 +505,24 @@ def build_custom_checkers(by_lang):
                          "static/templates/settings/bot-settings.handlebars",
                          "templates/zerver/email_log.html"])},
         {'pattern': "placeholder='[^{]",
-         'description': "`placeholder` value should be translatable."},
+         'description': "`placeholder` value should be translatable.",
+         'good_lines': ["<input placeholder='{foo()}'></input>"],
```

Comment:
This isn't what actual translation tags look like. Do `git grep placeholder` and take an actual example.

zulip.readthedocs.io/en/latest/translating.html is also a helpful resource.